src/analysis/retail/mage/frost/CHANGELOG.tsx

```diff
@@ -6,7 +6,7 @@ import { Earosselot, Sharrq, Sref, ToppleTheNun } from 'CONTRIBUTORS';
 
 // prettier-ignore
 export default [
-  change(date(2024, 20, 4), <>Improved <SpellLink spell={SPELLS.WINTERS_CHILL} /> analysis on guide. Setting guide as default view. </>, Earosselot),
```

Comment:
This is showing as 04/08/2025, which I don't believe to be true unless @earosselot is a timetraveller 🤣